code/modules/research/xenobiology/crossbreeding/_misc.dm

```diff
@@ -231,3 +231,200 @@ Slimecrossing Items
+	create_reagents(100, OPENCONTAINER)
+
+/obj/item/slime_iv/MouseDrop(mob/living/target)
+	. = ..()
+	if(!ishuman(usr) || !usr.canUseTopic(src, BE_CLOSE) || !isliving(target))
+		return
+
+	if(attached == target)
+		visible_message("<span class='warning'>[attached] is detached from [src].</span>")
+		attached = null
+		return
+
+	if(!target.has_dna())
+		to_chat(usr, "<span class='danger'>The drip beeps: Warning, incompatible creature!</span>")
+		return
+
+	if(Adjacent(target) && usr.Adjacent(target))
+		usr.visible_message("<span class='warning'>[usr] attaches [src] to [target].</span>", "<span class='notice'>You attach [src] to [target].</span>")
+		add_fingerprint(usr)
+		attached = target
+		START_PROCESSING(SSprocessing, src)
+
+/obj/item/slime_iv/process()
+	if(!attached)
+		return PROCESS_KILL
+
+	if(!(get_dist(src, attached) <= 1 && isturf(attached.loc)))
+		to_chat(attached, "<span class='userdanger'>The IV drip needle is ripped out of you!</span>")
+		attached.apply_damage(3, BRUTE, pick(BODY_ZONE_R_ARM, BODY_ZONE_L_ARM))
+		attached = null
+		return PROCESS_KILL
+
+	if(reagents.total_volume)
+		reagents.trans_to(attached, 5, method = INJECT, show_message = FALSE) //make reagents reacts, but don't spam messages
+		return
+
+/obj/item/slime_letter
+	name = "Slime letter"
+	icon = 'icons/obj/slimecrossing.dmi'
+	icon_state = "slime_letter"
+
+/obj/item/slime_letter/Initialize()
+	. = ..()
+	START_PROCESSING(SSprocessing,src)
+
+/obj/item/slime_letter/process()
+	var/humanfound = null
+	if(ishuman(loc))
+		humanfound = loc
+	if(!humanfound)
+		return
+	var/mob/living/carbon/human/human = humanfound
+	human.apply_status_effect(/datum/status_effect/lovers_hug)
+
+/obj/item/slime_veil
+	name = "Slime veil"
+	icon = 'icons/obj/slimecrossing.dmi'
+	icon_state = "slime_veil"
+	///who holds us
+	var/mob/wielder
+
+/obj/item/slime_veil/Initialize()
+	. = ..()
+	RegisterSignal(src, COMSIG_TWOHANDED_WIELD, .proc/on_wield)
+	RegisterSignal(src, COMSIG_TWOHANDED_UNWIELD, .proc/on_unwield)
+
+/obj/item/slime_veil/ComponentInitialize()
+	. = ..()
+	AddComponent(/datum/component/two_handed, force_unwielded=8, force_wielded=12)
+
+/obj/item/slime_veil/process()
+	if(!wielder)
+		return PROCESS_KILL
+	///you will always be barely visible
+	wielder.alpha = max(10, wielder.alpha - 25)
+
+
+/obj/item/slime_veil/proc/on_wield(datum/source,mob/user)
+	START_PROCESSING(SSprocessing,src)
+	RegisterSignal(user, COMSIG_MOVABLE_MOVED ,.proc/moved)
+	wielder = user
+
+/obj/item/slime_veil/proc/moved(datum/source)
+	var/atom/movable/moved = source
+	moved.alpha = 255
+
+/obj/item/slime_veil/proc/on_unwield(datum/source,mob/user)
+	STOP_PROCESSING(SSprocessing,src)
+	UnregisterSignal(user, COMSIG_MOVABLE_MOVED)
+	wielder = null
+
+/obj/item/kitchen/knife/slime
+	name = "slime knife"
+	color = "#00ff15"
+	force = 0
+
+/obj/item/claymore/slime
+	name = "slime claymore"
+	force = 18
+	color = "#00ff15"
+	block_chance = 10
+
+obj/item/tank/internals/emergency_oxygen/slime
```

Comment:
idk dude, this kinda already overlaps with the rebreather crossbreed item, which just straight up makes you not need to breathe